user_tools/src/spark_rapids_tools/cmdli/tools_cli.py

```diff
@@ -287,6 +288,33 @@ def train(self,
                          wrapper_options=train_args)
         tool_obj.launch()
 
+    def stats(self,
```

Comment:
Should `spark_rapids stats` fail if `qual_output` is not passed? We should probably have validator check for this.